docs/source/defining-shortcuts.md

```diff
@@ -55,6 +55,42 @@ The full list of available placeholders is available at {ref}`placeholders`.
 This is not using any customization options or advanced features. It's the bare minimum to make it
 work: a name, the command, and the target platforms.
 
+## Specifying different shortcut names for base and non-base environments
+
+If environments are supported, different naming schemes can be specified for installations into
+the base environment and non-base environments.
+To do this, the `name` property must be a dictionary with the keys `target_environment_is_base`
+and `target_environment_is_not_base` for installations into the base and non-base environment,
+respectively.
+
+
+The example below creates a shortcut called with the name "Launch Turtle" if installed into the
+base environment. If installed into an environment called, e.g., `turtle`, the name of the shortcut
+is "Launch Turtle (turtle)". This was the default behavior of `menuinst` version 1.
+
+```json
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "$id": "https://schemas.conda.io/menuinst-1.schema.json",
+  "menu_name": "Python {{ PY_VER }}",
+  "menu_items": [
+    {
+      "name": {
+        "target_environment_is_base": "Launch Turtle",
+        "target_environment_is_not_base": "Launch Turtle ({{ ENV_NAME }})"
```

Comment:
I'm thinking of non-conda implementations of the schema (where there's no "base" environment, like pixi or micromamba), and whether they'll have to deal with these semantics. They'll probably ignore `*_is_base`, but will have to handle `*_is_not_base`. 

In a way, it looks to me that `target_environment_is_not_base` is the default case, while `target_environment_is_base` seems to be a special case in the grand scheme of things.

Should we have this as `default` and `target_environment_is_base`, instead? I know I'm bikeshedding a bit, but I wanted to make sure that other angles are considered.

cc @wolfv for awareness.

Reply:
I think `default` is ambiguous and what's the most common use case for micromamba and pixi is not necessarily the use case for others. I would say that for `conda`, the base environment case is much more common since the console shortcut, e.g., is installed into the base environment via the installer.

I think explicit is better than implicit.